integration-tests/azure-keyvault/README.md

```diff
@@ -0,0 +1,163 @@
+# Azure Key Vault sample
+
+This is a sample application using the Quarkus Key Vault extension to create secret with `SecretClient` and `SecretAsyncClient` from Azure Key Vault.
+
+## Prerequisites
+
+To successfully run this sample, you need:
+
+* JDK 11+ installed with JAVA_HOME configured appropriately
+* Apache Maven 3.8.6+
+* Azure CLI and Azure subscription if the specific Azure services are required
+* Docker if you want to build the app as a native executable
+
+You also need to make sure the right version of dependencies are installed.
+
+### Use development iteration version
+
+By default, the sample depends on the development iteration version, which is `999-SNAPSHOT`. To install the development
+iteration version, you need to build it locally.
+
+```
+# Switch to the root directory of Quarkus Azure services extensions.
+# For example, if you are in the directory of quarkus-azure-services/integration-tests/azure-keyvault
+cd ../..
+
+# Install all Quarkus Azure services extensions locally.
+mvn clean install -DskipTests
+
+# Switch back to the directory of integration-tests/azure-keyvault
+cd integration-tests/azure-keyvault
+```
+
+### Use release version
+
+If you want to use the release version, you need to update the version of dependencies in the `pom.xml` file.
+
+First, you need to find out the latest release version of the Quarkus Azure services extensions
+from [releases](https://github.com/quarkiverse/quarkus-azure-services/releases), for example, `1.0.0`.
+
+Then, update the version of dependencies in the `pom.xml` file, for example:
+
+```xml
+
+<parent>
+    <groupId>io.quarkiverse.azureservices</groupId>
+    <artifactId>quarkus-azure-services-parent</artifactId>
+    <version>1.0.0</version>
+    <relativePath></relativePath>
+</parent>
+```
+
+## Preparing the Azure services
+
+You need to create an Azure Key Vault before running the sample application.
+
+### Logging into Azure
+
+Log into Azure and create a resource group for hosting the Key Vault to be created.
+
+```
+az login
+
+RESOURCE_GROUP_NAME=<resource-group-name>
+az group create \
+    --name ${RESOURCE_GROUP_NAME} \
+    --location eastus
+```
+
+### Creating Azure Key Vault
+
+Run the following commands to create an Azure Key Vault, set permission and export its endpoint as an environment
+variable.
+
+```
+KEY_VAULT_NAME=<unique-key-vault-name>
+az keyvault create --name ${KEY_VAULT_NAME} \
+    --resource-group ${RESOURCE_GROUP_NAME} \
+    --location eastus
```

Comment:
Mention the GitHub Actions workflow.

Reply:
See https://github.com/quarkiverse/quarkus-azure-services/commit/5fce42ce138dd83c3e4baf06b1a1b76ec8c82829.